src/app/token/[tokenId]/LinksMenu.tsx

```diff
@@ -0,0 +1,47 @@
+import { TokenInfoProps } from '@/pages/token/[tokenId]';
+import { useColorMode } from '@chakra-ui/react';
+import { Menu } from '@/ui/Menu';
+import { MenuButton } from '@/ui/MenuButton';
+import { Button } from '@/ui/Button';
+import { Icon } from '@/ui/Icon';
+import { BsChevronDown } from 'react-icons/bs';
+import { MenuList } from '@/ui/MenuList';
+import React from 'react';
+import { LinksGroup } from '@/app/token/[tokenId]/LinksGroup';
+import { MenuDivider } from '@/ui/MenuDivider';
+
+export function LinksMenu(props: { links: TokenInfoProps['extended']['links'] }) {
+  const colorMode = useColorMode().colorMode;
+  return (
+    <Menu>
+      <MenuButton
+        as={Button}
+        backgroundColor={colorMode === 'light' ? 'white' : 'transparent'}
+        variant={colorMode === 'light' ? undefined : 'outline'}
+        color={'textTitle.light'}
+        rightIcon={<Icon as={BsChevronDown} size="11px" color={'textCaption.light'} />}
+        _hover={{ backgroundColor: colorMode === 'light' ? 'white' : 'transparent' }}
+      >
+        Links
+      </MenuButton>
+      <MenuList>
+        {[
+          { title: 'Websites', links: props.links.websites },
```

Comment:
All of these titles are just the capitalized link key, so you could map through the keys of the object and render the title as the capitalized key (and pass the links the same)? Just an idea, feel free to ignore of course